src/_locales/es/messages.json

```diff
@@ -366,11 +366,11 @@
         "description": "Dropdown control setting on the Tracking Domains options page tab."
     },
     "disabled_for_these_domains": {
-        "message": "<p>Privacy Badger está desactivado en los siguientes sitios. Esto significa que Privacy Badger no bloqueará nada cuando usted visite los sitios listados aquí.</p><p>Si cree que Privacy Badger está rompiendo una página, puede escribir el dominio de esa página en el cuadro de abajo y hacer clic en el botón \"Añadir dominio\".</p><p>Alternativamente, cuando ya tenga seleccionada la pestaña de la página, simplemente haga clic en el botón de Privacy Badger en la barra de herramientas del navegador y luego haga clic en el botón \"Deshabilitar\".</p>",
+        "message": "<p>Privacy Badger está desactivado en los siguientes sitios. Esto significa que Privacy Badger no bloquea nada cuando usted visita los sitios listados aquí.</p><p>Si cree que Privacy Badger está rompiendo una página, puede escribir el dominio de dicha página en el cuadro de abajo y hacer clic en el botón \"Añadir el dominio\".</p><p>Alternativamente, cuando ya tenga seleccionada la pestaña de la página, simplemente haga clic en el botón de Privacy Badger en la barra de herramientas del navegador y luego haga clic en el botón \"Deshabilitar\".</p>",
```

Comment:
Should we switch the "" quotes here to «» like in the `non_tracker_tip` message?

Reply:
I think so. «» are the Spanish quotes and as far as I know the preference order in Spanish is «», "", ''. The first is for the outer quote and the other are used when you have a nested quote. Shall I change them?

Reply:
Yes please.